packages/flutter_test/lib/src/test_default_binary_messenger.dart

```diff
@@ -8,6 +8,11 @@ import 'dart:ui' as ui;
 import 'package:fake_async/fake_async.dart';
 import 'package:flutter/services.dart';
 
+/// A function which takes the name of the method channel, it's handler,
+/// platform message and asynchronously returns an encoded response.
```

Comment:
document in what cases handler could be null.

Reply:
Can you please address this comment?

Reply:
Ah, sorry, I thought this comment was on the allMessagesHandler and not the type, and thought it was addressed. Added now :)